.circleci/config.yml

```diff
@@ -14,6 +14,12 @@ tags: &tags
   ]
 
 jobs:
+  check-license:
```

Comment:
There's another part that needs to be added to the CI script at the bottom of the file. It looks like this:

```
      - check-license:
          filters:
            tags:
              only: /.*/
```